src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -0,0 +1,166 @@
+                })}
+                data-testid="task-card"
+            >
+                <Comment
+                    created_at={created_at}
+                    created_by={created_by.target}
+                    currentUser={currentUser}
+                    error={error}
+                    id={id}
+                    inlineDeleteMessage={messages.taskDeletePrompt}
+                    isPending={isPending}
+                    onDelete={onDelete}
+                    onEdit={onEdit}
+                    permissions={taskPermissions}
+                    tagged_message={name}
+                    translatedTaggedMessage={translatedTaggedMessage}
+                    translations={translations}
+                    getAvatarUrl={getAvatarUrl}
+                    getUserProfileUrl={getUserProfileUrl}
+                    mentionSelectorContacts={mentionSelectorContacts}
+                    getMentionWithQuery={getMentionWithQuery}
+                />
+                <div className="bcs-task-assignment-container">
+                    {due_at ? (
+                        <div
+                            className={classNames('bcs-task-due-date', {
+                                'bcs-task-overdue': isOverdue,
+                            })}
+                            data-testid="task-due-date"
+                        >
+                            <FormattedMessage {...messages.taskDueDate} />
+                            <FormattedTime value={due_at} day="numeric" month="short" year="numeric" />
+                        </div>
+                    ) : null}
+                    <div className="bcs-task-assignments">
+                        {assigned_to && assigned_to.entries
+                            ? assigned_to.entries
+                                  .map(fillUserPlaceholder)
+                                  .slice(0, MAX_AVATARS)
+                                  .map(({ id: assignmentId, target, status: assigneeStatus }) => {
+                                      return (
+                                          <AssigneeStatus
+                                              key={assignmentId}
+                                              status={assigneeStatus}
+                                              user={target}
+                                              getAvatarUrl={getAvatarUrl}
+                                              data-testid="task-assignment-status"
+                                          />
+                                      );
+                                  })
+                            : null}
+                        {hiddenAssigneeCount > 0 ? (
+                            <span className="bcs-task-assignment-avatar bcs-task-assignment-count">
+                                +{hiddenAssigneeCount}
+                            </span>
+                        ) : null}
+                    </div>
+                    {status === TASK_NEW_INCOMPLETE &&
```

Comment:
Can we pull this set of conditions into a variable to help explain what "mode" we expect to be in?